README.md

```diff
@@ -1,4 +1,50 @@
+
+**A review will either result in PING raising issues, raising serious concerns, or raising neither.**
+
+Review is not a sign-off or authorization step — it involves substantial engagement with the design and implementation of the specification as well as its goals and likely impact on the web platform. PING recommends that specification authors ask PING to start a review as soon as they have the broad outline of a specification, and no less than once every six months during a specification's lifespan. Many privacy issues can be anticipated at this early stage. Learning about these potential issues allows working groups and specification authors to implement solutions smoothly and thoughtfully. This should lead to fewer issues raised during review.
+
+### Raising Issues
+
+**Issues** are specific privacy drawbacks with the specification which must be addressed before the specification becomes a recommendation. When raising issues, PING will endeavor to articulate posssible technical approaches which mitigate those issues.
+
+When issue are raised, PING will open GitHub issues in the specification's repository describing the concern and potential options for solutions. If the specification's working group do not use GitHub issues, they should specify an alternative issue tracking location in the review request, and PING will use that instead. PING will also list each issue opened in the privacy review request.
+
+Specification authors should work with PING to resolve these issues. Once all issues have been resolved to the satisfaction of the authors and of PING, PING will resolve the privacy review request.
+
+### Serious Concerns
+
+**Serious Concerns** are fundamental problems with the approach or goals of the specification. These are unlikely to have straightforward technical remediations. They require fundamentally re-thinking or re-architecting the specification. This should be rare.
+
+When there are serious concerns with a specification, PING will outline them in a response to the privacy review request. Specification authors should work with PING to understand these concerns and re-architect the specification, which may require re-chartering. When the specification has been re-thought in this way, specification authors should open a new privacy review request for the new specification
+
+## Review Status
+
+Every specification is therefore in one of four possible states:
+
+1. This specification has been reviewed by PING. There are no issues or concerns.
+2. This specification has been reviewed by PING. There are issues to resolve.
+3. This specification has been reviewed by PING. PING has serious concerns! [There may also be issues to resolve.]
+4. This specification has not been reviewed by PING.
+
+This status should be indicated at the top of all specification documents, alongside the similar status from other horizontal review groups.
+
+Specifications which are not in state #1 are not eligible to proceed to recommendation status.
```

Comment:
All for more and more thorough horizontal reviews, but I don't believe this statement is true. 

If there are disagreements between groups, it's an input and a data point for the process but not a "not eligible" blocker. Case in point, webperf WG may express concerns about potential performance implications of the feature, but that does not mean that it makes in not eligible to proceed to recommendation status.